docs/embedding.md

```diff
@@ -428,47 +437,101 @@ _These settings are NOT configurable via the Endpoint Editor in Cognigy.AI_
+| audioContent              | `Object` | [See Audio Content](#audio-content)                | Audio related aria labels                                                                                                |
+| fileContent               | `Object` | [See File content](#file-content)                  | File content related aria labels                                                                                         |
+| messageHeader             | `Object` | [See Message Header](#message-header)              | Labels for message headers in conversation                                                                               |
+| audioTimeRemaining        | string   | "{time} remaining"                                 | This text is used by aria-valuetext attribute of input element indicating the remaning time left in the Audio playback   |
+
+##### Image Content
+
+Image content related text announcements for image elements
+| Name | Type | Default | Description |
+| --------------- | ------ | ------- | ----------- |
+| downloadable | string | "An image with download option. {altTextLabel}" | This text indicates an image which is downloadable. `altText` is based on the image metadata |
+| nonDownloadable | string | "An image. {altTextLabel}" | This text indicates an image which is not downloadable.`altText` is based on the image metadata |
+
+##### Video Content
+
+Video content related text announcements for video elements
+| Name | Type | Default | Description |
+| ---------------------------- | ------ | ------- | ----------- |
+| withTranscriptAndCaptions | string | "A video with transcript and captions." | This text is used by video elements with transcript and captions |
+| withTranscript | string | "A video with transcript." | This text is used by video elements with transcript |
+| withCaptions | string | "A video with captions." | This text is used by video elements with only captions |
+| withoutTranscriptAndCaptions | string | "A video message." | This text is used by video elements without transcript and captions |
+
+##### Audio Content
+
+Audio content related text announcements for audio elements
+| Name | Type | Default | Description |
+| ----------------- | ------ | ------- | ----------- |
+| withTranscript | string | "An audio message with transcript" | This text is used by audio element with transcript |
+| withoutTranscript | string | "An audio message." | This text is used by audio element without transcript |
+
+##### File Content
+
+File content related text announcements for file attachments
+| Name | Type | Default | Description |
+| ------------- | ------ | ------- | ----------- |
+| singleFile | string | "{text}. A {type} named '{fileName}' with size {sizeLabel}." | This text is for single file attachments |
```

Comment:
We could leave out the text key from here. Also, there is a small issue with 'type'. Type can either be 'image' or 'file' (hardcoded in English). So, this cannot be translated

Reply:
Still not resolved